packages/pwa/node.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = function installServiceWorke() {
+  return new Promise(function() {});
```

Comment:
Should add a comment to explain that this promise intentionally never resolves. (Can't register a Service Worker in Node.js environment).